pr2_description/robots/pr2.urdf.xacro

```diff
@@ -24,7 +24,6 @@
   <xacro:include filename="$(find pr2_description)/urdf/base_v0/base.urdf.xacro" />
   <!-- Head sensors -->
   <xacro:include filename="$(find pr2_description)/urdf/sensors/head_sensor_package.urdf.xacro" />
-  <xacro:include filename="$(find pr2_description)/urdf/sensors/kinect_prosilica_camera.urdf.xacro" />
```

Comment:
Yeah, in the original file there's a joint and link for the prosilica camera attached to the Kinect. I never removed the include, just moved it inside an unless statement at the bottom for people using the original Kinect